src/rust/build.rs

```diff
@@ -42,5 +59,14 @@ const ALLOWLIST_FUNCTIONS: &[&str] = &[
     "printdata",
     "writercwtdata",
 ];
+
+#[cfg(feature = "hardsubx_ocr")]
+const ALLOWLIST_FUNCTIONS_OCR_HARDSUBX: &[&str] =
+    &["edit_distance", "convert_pts_to_.*", "av_rescale_q"];
+
```

Comment:
Could you change this and other lists to something like, 
``` rust
let mut allowlist_functions = vec![
    ".*(?i)_?dtvcc_.*",
    "get_visible_.*",
    "get_fts",
    "printdata",
    "writercwtdata",
];

#[cfg(feature = "hardsubx_ocr")]
allowlist_functions.extend_from_slice(&["edit_distance", "convert_pts_to_.*", "av_rescale_q"]);

```

Then there will be no need for another loop just to add hardsubx stuff

Reply:
Would have to move the definitions inside the main function then. They won't be globals anymore. 

That's not an issue right?

Reply:
Also @PunitLodha  this leads to mutability not being required if the flags are off and that leads to a warning and fails the build 

I will recommend keeping the original version

Reply:
That is due to the feature not being selected. Could you create a new PR and add the `--all-features` flag to the clippy command in the `format_rust` workflow